### DISCUSSION TITLE
Is there a way to extend the uptime of a propposal to 1 year?

### DISCUSSION THREAD

**Discussion by Lorenzobattistela** [snapshot-labs/snapshot#2692] [Q&A] [answered]
Me and my partner are using snapshot for voting proposals in our DAO. The thing is, our DAO is a learn-to-earn model, and we were thinking of using voting in our snapshot as a mission (to earn).

For this to work, we have to test how snapshot works for a while, and also have a durable proposal so that students can complete the task for a long time. 

That said, my question is if it is possible to set the duration of the proposal to more than 1 week (that appears to be the standard), and how to do it. 

Also I wanted to know if it is only the admin of the space that can extend the duration of a proposal.

Thanks in advance.

**Comment by metrox-eth** [comment, marked as answer]
Yes voting period can be set for years, it can be set when you create the proposal or in your settings.
